guides/common/modules/ref_migrating-from-foreman_webhooks.adoc

```diff
@@ -0,0 +1,12 @@
+[id="migrating-webhooks_{context}"]
+= Migration to Foreman Webhooks
+
+The legacy `foreman_hooks` provides full access to model objects that the webhooks plugin does not intentionally provide.
+
+The scope of what is available is limited by the safemode and all objects and macros are both subject to API stability promise and are fully documented.
+
+The number of events triggered by webhooks is substantially less than with `foreman_hooks`.
+
+The webhooks plugin is designed to be safe, hooks are processed asynchronously so there is minimal risk of tampering with internals of the system. It is not possible to migrate from `foreman_hooks` without creating payloads for each individual webhook script.
```

Comment:
```suggestion
{Project} webhooks are designed to be safe. 
Webhooks are processed asynchronously so there is minimal risk of tampering with internals of the system. 
It is not possible to migrate from `foreman_hooks` without creating payloads for each individual webhook script.
```

A colon or a full stop is better than the comma here, imo. 
And I moved onto a newline if using comma.